3-networks/envs/dev/main.tf

```diff
@@ -27,15 +27,15 @@ locals {
 *****************************************/
 
 data "google_projects" "restricted_host_project" {
-  filter = "labels.application_name=restricted-shared-vpc-host-${local.env}  lifecycleState=ACTIVE"
+  filter = "labels.application_name=restricted-shared-vpc-host-${local.env}"
```

Comment:
I think these have been reverted from the changes that @daniel-cit made, can we change it back?